pom.xml

```diff
@@ -64,7 +64,7 @@
     <properties>
         <revision>2.41</revision>
         <changelist>-SNAPSHOT</changelist>
-        <jenkins.version>2.176.4</jenkins.version>
+        <jenkins.version>2.264-SNAPSHOT</jenkins.version>
```

Comment:
Why?

Reply:
Changes in Jenkins core https://github.com/jenkinsci/jenkins/pull/5023#pullrequestreview-514587286

Reply:
Use incremental versions if you want CI.